tools/driver/cling.cpp

```diff
@@ -106,6 +107,31 @@ int main( int argc, char **argv ) {
     Interp.loadFile(Lib);
 
   cling::UserInterface Ui(Interp);
+
+  // Get the home directory
+  std::string HomeDir;
+#ifdef __unix__
+  HomeDir = getenv("HOME");
+  HomeDir += "/";
+#elif defined(_WIN32)
+  HomeDir = getenv("DRIVE");
+  const char *HomePath = getenv("HOMEPATH");
+  HomeDir += HomePath;
+  HomeDir += "\\";
```

Comment:
You can maybe use the `%USERPROFILE%` environment variable instead?
```
   HomeDir = getenv("USERPROFILE");
```

Reply:
Alright